.github/workflows/ci.yaml

```diff
@@ -48,6 +48,7 @@ jobs:
       CC: deny_c
       RUST_CHANNEL: "${{ needs.changes.outputs.proc_macros == 'true' && 'nightly' || 'stable' }}"
       USE_SYSROOT_ABI: "${{ needs.changes.outputs.proc_macros == 'true' && '--features sysroot-abi' || '' }}"
+      RUSTUP_RUSTC_DEV: "${{ needs.changes.outputs.proc_macros == 'true' && 'rustc-dev' || '' }}"
```

Comment:
We want this, right?

Reply:
No we shouldn't need this 🤔 our CI should not rely on the rustc crates